package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "is-my-json-valid",
-  "version": "2.16.1",
+  "version": "2.17.1",
```

Comment:
A version change for the `false` and `true` schemas should probably be a breaking `v3` change. It will break any schemas relying on the bug.